src/pages/sigcse2024/index.tsx

```diff
@@ -46,14 +46,28 @@ export default function SIGCSE2024() {
             </p>
             <p>
               Currently, PrairieLearn is actively adopted in over 800 courses
-              across 20 universities. In this affiliated event, instructors from
-              these courses will showcase the innovative online assessments
-              they’ve created using PrairieLearn, and they are eager to share
-              these resources with the broader instructor community. Presenters
-              will also be happy to discuss their experience authoring these
-              assessments.
+              across 20 universities.{" "}
+              <strong>
+                In this affiliated event, instructors from these courses will
+                showcase the innovative online assessments they’ve created using
+                PrairieLearn, and they are eager to share these resources with
+                the broader instructor community.
+              </strong>{" "}
+              Presenters will also be happy to discuss their experience
+              authoring these assessments.
             </p>
 
+            <div className={classnames("container-fluid py-2")}>
+              <div className="d-flex flex-row justify-content-center flex-wrap">
+                <Link
+                  href="sigcse2024/submit-questions"
```

Comment:
Nit, we prefer absolute URLs where possible.

```suggestion
                  href="/sigcse2024/submit-questions"
```